### ISSUE TITLE
Add DE-9IM graphics to Web Documentation

### ISSUE THREAD

**Issue by DenisCarriere** [Turfjs/turf#1140]
## Add DE-9IM graphics to Web Documentation

I was attempting to explain what `@turf/boolean-disjoint` was to someone and even I got confused on what that module was meant to do, I then decided to go to ESRI's docs and look at the **disjoint** graphic for more help.

We should try to include at least 1 info graphic to each module (optional) which helps better understand what the module does in a conceptual way.

http://edndoc.esri.com/arcsde/9.0/general_topics/understand_spatial_relations.htm

These infographics are very useful

**equal**

![image](https://user-images.githubusercontent.com/550895/33394746-2a97d3f4-d4f8-11e7-9ee0-870666c41527.png)

**disjoint**

![image](https://user-images.githubusercontent.com/550895/33394751-2f8e2f16-d4f8-11e7-8e12-ae33eac26b36.png)

**Comment by rowanwins** [comment]
Yeah that would be nice @DenisCarriere , there may be some newer graphics somewhere on the ESRI site so might be worth a quick dig around.

**Comment by stebogit** [comment]
👍 👍

**Comment by DenisCarriere** [comment]
👍 We just need to find a consistent naming convention for these infographics.

Maybe we can include a collection of images to create a sort of Photo Gallery of maps/graphics which derive from that particular Module.

## Image Folder Structure

- `images/*.png`
- `images/*.jpg`

**Example:** `./packages/turf-center/images/infographic.jpg`

**Comment by rowanwins** [comment]
Do we put in the images in the turf-www repo ?

**Comment by DenisCarriere** [comment]
No we should keep them in the module repo, that way we can also include those images when we are using [`scripts/generate-readmes`](https://github.com/Turfjs/turf/blob/master/scripts/generate-readmes)

The `turf-www` repo already has `turf` built inside, so we can reach those images through the main Turf folder.

I've done the same thing for the [`documentation.yml` config](https://github.com/Turfjs/turf-www/pull/128/files#diff-66599dd3d735c957a4e336cc8211f6edR19).

**Comment by rowanwins** [comment]
yeah ok, no worries. Well if that's the case then I reckon your proposed above looks good.

Slightly newer Esri image here
http://desktop.arcgis.com/en/arcmap/10.3/manage-data/using-sql-with-gdbs/relational-functions-for-st-geometry.htm
Although not sure if we can steal them due to licensing...

The postgis images are a bit more complicated because they are split into lots of portions.
https://postgis.net/docs/ST_Contains.html

That said I probably whip something up

**Comment by DenisCarriere** [comment]
We can create our own diagrams to prevent any ESRI image licensing complications.

I'll set up the `scripts/generate-readmes` to work with `images/*.(png|jpg)`

**Comment by muziejus** [comment]
I can spruce up some of the images from Esri, etc., but I think I'm generally a bit better off building on something that already exists. 

It would make sense to come up w/ a standardized visual language, also, much like Esri has for its various geoprocessing images:

![Dissolve](http://desktop.arcgis.com/en/arcmap/10.3/tools/coverage-toolbox/GUID-E49A133D-83D0-4217-8319-DCAAD61F3157-web.gif)

![Eliminate Sliver Polygons](http://resources.esri.com/help/9.3/arcgisengine/java/gp_toolref/data_management_tools/eliminate_esri.gif)

![Clip](http://resources.esri.com/help/9.3/arcgisengine/java/gp_toolref/analysis_tools/clip_esri.gif)

**Comment by rowanwins** [comment]
First attempt at a unified style. Graphic on the left, and some brand guidance on the right.

![turfgraphics](https://user-images.githubusercontent.com/6735870/33936125-8c1b5208-e052-11e7-9b2b-114dc367c058.png)

**Comment by DenisCarriere** [comment]
Very cool! I like the style guide

**Comment by DenisCarriere** [comment]
We should try to find a clever spot for this on the website, maybe we could add it to a contribution section.

**Comment by ppKrauss** [comment]
Hi, I was maintaining the Wikipedia page [DE-9IM](https://en.wikipedia.org/wiki/DE-9IM), perhaps we can  join forces to maintain didactic content for Turf and for Wikipedia... The main problem as you see are the good illustrations.

DE-9IM Wikpedia's article is stable and is is complete in terms of subjects and links... Now needs a big review do be "accessible". Even for people  with a mathematics background, they found to hard to follow.  All here are invited to review or suggest enhancements there. 

------

## Joining forces with Wikipedia

Many good guides and softare documentation are interlinking Wikipedia. For example the [DE-9IM  illustration of PostGIS](https://postgis.net/docs/images/de9im06.png) was used by Wikipedia, and PostGIS linked Wikipedia's DE-9IM article [here](https://postgis.net/docs/ST_Relate.html) and [here](https://postgis.net/docs/using_postgis_dbmanagement.html#DE-9IM).

**Comment by rowanwins** [comment]
Hi @ppKrauss 

Thanks for your work on the wikipedia page, it looks great! It's certainly a tricky subject to get your head around and it seems like there are lots of little pitfalls.

As you can see this issue hasn't been touched in quite a while as there have been other priorities that have taken my attention. 

I'd be keen to support creating some high quality graphics, perhaps we can create a github repo and create them as SVG and then export them as png for use. Half the battle is setting up a half-decent template, but once we had something it could be a collaborative effort to create them using something like [InkScape](https://inkscape.org/). Although there are probably some challenges in that certain projects will want to include certain details on the grahpics and others wont, so I'm not sure how reusable they would really be in reality... Although theoretically with an svg in inkscape you can export only the bits of the artboard that you are interested in.

Thoughts?
